runtime/verbose/errormessageframeworkrtv.c

```diff
@@ -439,7 +439,7 @@ getBCVDataType(J9BytecodeVerificationData* verifyData, MethodContextInfo* method
 		if (J9_ARE_ALL_BITS_SET(bcvType, BCV_PRIMITIVE_VALUETYPE)) {
 		   typeNameIndex = CFR_STACKMAP_TYPE_PRIMITIVE_OBJECT;
```

Comment:
While you're here, can you please fix the indentation of this line.

Reply:
fixed in [89dde4a](https://github.com/eclipse-openj9/openj9/commit/89dde4a1efae9e5a1c1a1991631067030591c6ee)